plugins/generic/externalFeed/controllers/grid/ExternalFeedGridHandler.inc.php

```diff
@@ -0,0 +1,240 @@
+<?php
+/**
+ * @file controllers/grid/ExternalFeedGridHandler.inc.php
+ *
+ * Copyright (c) 2014-2016 Simon Fraser University Library
+ * Copyright (c) 2003-2016 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class ExternalFeedGridHandler
+ * @ingroup controllers_grid_externalFeed
+ *
+ * @brief Handle external feeds grid requests.
+ */
+import('lib.pkp.classes.controllers.grid.GridHandler');
+import('plugins.generic.externalFeed.controllers.grid.ExternalFeedGridRow');
+import('plugins.generic.externalFeed.controllers.grid.ExternalFeedGridCellProvider');
+
+class ExternalFeedGridHandler extends GridHandler {
+	/** @var ExternalFeedPlugin The external feeds plugin */
+	static $plugin;

```

Comment:
Ah, I see. This could probably use some cleanup but better to do it consistently with other plugins -- in other words, don't worry about it for now :)